system/behavior.lua

```diff
@@ -65,25 +68,50 @@ function Behavior:Initialize(isReload)
   self:ReportLoadedBehaviors()
 end
 
+function Behavior:DecideBestSpecialization()
```

Comment:
This function was moved to WoWActivePlayer:Specialization()

Remove this function and use `Me:Specialization` instead